### DISCUSSION TITLE
Color gap in Paste Evergreen Theme

### DISCUSSION THREAD

**Discussion by char-mainewang** [twilio-labs/paste#3135] [Help: Figma] [answered]
Found a color gap between Paste Twilio theme and Paste Evergreen Theme while trying to swap colors 

<img width="1278" alt="image" src="https://user-images.githubusercontent.com/87334135/228664698-0c8e5c0c-b681-4b43-8f91-a2359fec1269.png">

**Comment by loreina** [comment]
Hi @char-mainewang,

Thanks for catching this! Looks like the `color-text-user` token doesn't exist in the Paste Evergreen Theme. I've created a ticket to our backlog to fix this and someone on our team will follow up here once that fix is in.

Since the token doesn't exist, we'll likely just use the same alias as in the Paste Twilio Theme so you can safely swap themes without worrying about any breaking changes around that token. Thanks!!

**Comment by serifluous** [comment, marked as answer]
Hi @char-mainewang , we just published the fix for this in the Paste Evergreen theme Figma library. Thanks again for catching this!
